packages/opentelemetry-cloud-trace-exporter/src/transform.ts

```diff
@@ -77,6 +75,13 @@ export function getReadableSpanTransformer(
   };
 }
 
+function transformTime(time: ot.HrTime): Timestamp {
```

Comment:
How does the `hrTimeToTimeStamp()` differ from this?

Reply:
`hrTimeToTimeStamp()` returns a timestamp string, which is used on the REST endpoint. The grpc endpoint takes a [google.Protobuf.WellKnownTypes.Timestamp](https://cloud.google.com/trace/docs/reference/v2/rpc/google.devtools.cloudtrace.v2#span)